core/src/main/java/com/yahoo/ycsb/Client.java

```diff
@@ -553,6 +668,20 @@ private static void exportMeasurements(Properties props, int opcount, long runti
       exporter.write("OVERALL", "RunTime(ms)", runtime);
       double throughput = 1000.0 * (opcount) / (runtime);
       exporter.write("OVERALL", "Throughput(ops/sec)", throughput);
+      
+      boolean trackJVMStats = props.getProperty(Measurements.MEASUREMENT_TRACK_JVM_PROPERTY, 
```

Comment:
I'd like to dump a few stats at the end (particularly around GC) even if the user doesn't run the status thread (as it's disabled by default). I'll move this into a static (ugh) variable so we just parse it once.